staging/cert-manager-setup/templates/v1upgrade.yaml

```diff
@@ -0,0 +1,113 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: cert-manager-upgrade-backup
+  annotations:
+    "helm.sh/hook": pre-install
+    "helm.sh/hook-weight": "-10"
+    "helm.sh/hook-delete-policy": hook-succeeded,before-hook-creation
+spec:
+  template:
+    spec:
+      serviceAccountName: cert-manager-upgrade
+      containers:
+        - name: cert-manager-upgrade
+          image: "jaredrodriguez/cert-manager-upgrade:v0.1.0"
+          command: ["/cert-manager-upgrade", "backup"]
+      restartPolicy: Never
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: cert-manager-upgrade-restore
+  annotations:
+    "helm.sh/hook": post-install, post-upgrade
+    "helm.sh/hook-weight": "10"
```

Comment:
upgrade restore? So this does what? and wis the hook-weight correct?

Reply:
yeah, that name is a bit off.. the program is cert-manager-upgrade .. the command is restore. The weight is correct since I want to make absolutely sure this runs last, and ensure that anyone that adds hooks down the road knows that this should run last.